test-gateway-storage.md

```diff
@@ -0,0 +1,97 @@
+# Gateway Contract Storage Issue Test
```

Comment:
move to somewhere else (maybe alongside with the script?) or remove if leftover (probably belongs better in the GH issue)